package.json

```diff
@@ -4,7 +4,7 @@
   "description": "The libp2p build (module) used by js-ipfs on Node.js",
   "main": "src/index.js",
   "scripts": {
-    "test": "aegir-test node",
+    "test": "npm run test:muxer:spdy && npm run test:muxer:multiplex",
```

Comment:
@VictorBjelkholm adding both muxer tests to run now that we have 2 fully working muxers \o/